### ISSUE TITLE
'pip install hermit' does not install zbar dependency

### ISSUE THREAD

**Issue by kirillkovalenko** [unchained-capital/hermit#18]
On a fresh Ubuntu 19.10 I did install hermit with 'pip3 install hermit' and I'm trying to launch it. It crashes with 
```
raise ImportError('Unable to find zbar shared library')
```

**Comment by destrys** [comment]
Yup, zbar is a system dependency, not something pip will install.
In the Makefile you'll see it in `system-dependencies`, but it would be nice to make this explicit in the README and a better Error message.

**Comment by Stadicus** [comment]
The quick start is a bit short on details. For me on Ubuntu, instead of just `$ pip install hermit`, it should be more like:
```
# system dependencies
sudo apt install python3 python3-pip libzbar0        

# avoids python2-usage, uses recommended way of invoking pip
python3 -m pip install hermit
```

This does not take the creation of a `venv` into account, so cloning the repository and just running `make` is probably still the easier and more stable way to do this.
